docs/content/en/docs/resources/opensearch/_index.md

```diff
@@ -0,0 +1,177 @@
+---
+title: "OpenSearch"
+linkTitle: "OpenSearch"
+weight: 45
+---
+
+OpenSearch® is an open source search and analytics suite including search engine, NoSQL document database, and visualization interface. OpenSearch offers a distributed, full-text search engine based on Apache Lucene® with a RESTful API interface and support for JSON documents.
+
+> Before going through this guide, make sure you have a [Kubernetes cluster](../../installation/prerequisites/) with the [operator installed](../../installation/) and a [Kubernetes Secret with an Aiven authentication token](../../authentication/).
+
+## Creating an OpenSearch instance
+
+1. Create a file named `os-sample.yaml`, and add the following content: 
+
+```yaml
+apiVersion: aiven.io/v1alpha1
+kind: OpenSearch
+metadata:
+  name: os-sample
+spec:
+  # gets the authentication token from the `aiven-token` Secret
+  authSecretRef:
+    name: aiven-token
+    key: token
+
+  # outputs the OpenSearch connection on the `os-secret` Secret
+  connInfoSecretTarget:
+    name: os-secret
+
+  # add your Project name here
+  project: <your-project-name>
+
+  # cloud provider and plan of your choice
+  # you can check all of the possibilities here https://aiven.io/pricing
+  cloudName: google-europe-west1
+  plan: startup-4
+
+  # general Aiven configuration
+  maintenanceWindowDow: friday
+  maintenanceWindowTime: 23:00:00
+```
+
+2. Create the service by applying the configuration:
+
+```bash
+$ kubectl apply -f os-sample.yaml 
+```
+
+3. Review the resource you created with this command:
+
+```bash
+$ kubectl describe opensearch.aiven.io os-sample
+```
+
+The output is similar to the following:
+
+```bash
+...
+Status:
+  Conditions:
+    Last Transition Time:  2023-01-19T14:41:43Z
+    Message:               Instance was created or update on Aiven side
+    Reason:                Created
+    Status:                True
+    Type:                  Initialized
+    Last Transition Time:  2023-01-19T14:41:43Z
+    Message:               Instance was created or update on Aiven side, status remains unknown
+    Reason:                Created
+    Status:                Unknown
+    Type:                  Running
+  State:                   REBUILDING
+...
+```
```

Comment:
The `get` command doesn't actually show you plan, because it is a Aiven domain thing. Replaced with `describe`.